src/instructlab/model/serve.py

```diff
@@ -20,17 +19,23 @@
 @click.command()
 @click.option(
     "--model-path",
-    type=click.Path(),
-    default=config.DEFAULT_MODEL_PATH,
-    show_default=True,
+    cls=clickext.ConfigOption,
+    type=click.Path(path_type=pathlib.Path),
+    required=True,  # default from config
```

Comment:
Why is the comment needed?

Reply:
For humans. Or can you tell from the code that the default is coming from a mandatory field in `config.yaml`?

Reply:
I'm human, I guess I wasn't sure how to read it :)